MapboxNavigation/RouteMapViewController.swift

```diff
@@ -258,6 +260,34 @@ extension RouteMapViewController: NavigationMapViewDelegate {
             return defaultReturn
         }
         
+        if let userLocation = mapView.userLocation, routeController.showCurrentWayNameLabel == true {
+            
+            // Somehow check the current style has mapbox streets
+            //
+            //let streets = MGLSource(identifier: "mapbox://mapbox-streets-v7")
+            //guard mapView.style!.sources.contains(streets) else {
+            //  return
+            //}
+            
+            let userPuck = mapView.view(for: userLocation)
+            if let userPuck = userPuck {
+                let features = mapView.visibleFeatures(in: userPuck.frame)
+                
+                for layer in features {
+                    
+                    // TODO: check for acceptable classes
+                    // if let feature = layer as? MGLPolylineFeature,
+                    
+                    // TODO: Localize
+                    if let name = layer.attribute(forKey: "name") as? String {
```

Comment:
Should this match the map’s language or the UI language? Should we [automatically localize the map](https://github.com/mapbox/mapbox-gl-native/blob/8eb23cbac8a6c5821dc5935e27689700216c3f1a/platform/macos/app/MapDocument.m#L340-L387) to match the UI language (mapbox/ios-sdk-examples#36)?